src/components/new-safe/create/steps/ReviewStep/index.tsx

```diff
@@ -131,9 +131,8 @@ const ReviewStep = ({ data, onSubmit, onBack, setStep }: StepRenderProps<NewSafe
   const totalFee =
     gasLimit && maxFeePerGas
       ? formatVisualAmount(
-          maxFeePerGas +
-            // maxPriorityFeePerGas is undefined if EIP-1559 disabled
-            (maxPriorityFeePerGas || 0n) * gasLimit,
+          // maxPriorityFeePerGas is undefined if EIP-1559 disabled
+          (maxFeePerGas + (maxPriorityFeePerGas || 0n)) * gasLimit,
```

Comment:
Let's add a small unit test that would have picked this up.

Reply:
Extracted it into a helper function and wrote some tests. Also notice we do the same calculation for tx creation but don't take the maxPrioFee into account there. I added a TODO because the expected type is still BigNumber which wouldn't work with the new helper function and the usage of `Partial` somehow returns `any` as a type which is another issue.